cfpb/cfgov-refresh
# Modifying footer to add the Administrative Adjudication link

Modifying footer to add the Administrative Adjudication link

## Changes

- Modified `cfgov/jinja2/v1/_includes/organisms/footer.html` to add the `Administrative Adjudication link`. 

## Testing

- Add the `OAA_FOOTER_LINK` flag via Wagtail.
- Visit the site and observe that the footer link works as expected (will need latest dump).

## Screenshots

- 
<img width="1261" alt="screen shot 2017-03-30 at 11 29 46 am" src="https://cloud.githubusercontent.com/assets/1696212/24512659/32a5fa7e-153d-11e7-95ef-b7e5227c060c.png">


## Notes

- Slug will be `/administrative-adjudication-proceedings`.

## Todos

-

## Checklist
